imap/mboxlist.c

```diff
@@ -242,6 +242,9 @@ static struct dlist *mboxlist_entry_dlist(const char *dbname,
     if (mbentry->uniqueid)
         dlist_setatom(dl, "I", mbentry->uniqueid);
 
+    if (mbentry->root_uid)
```

Comment:
I know it's kind of obnoxiously long, but I'd avoid `uid` and either say "root_uniqueid" or "root_mailboxid" with an end-goal of renaming every "uniqueid" to "mailboxid", the RFC8474 blessed name.